api/android/api/src/main/java/org/nnsuite/nnstreamer/SingleShot.java

```diff
@@ -168,6 +191,27 @@ public void setTimeout(int timeout) {
         }
     }
 
+    /**
+     * Sets the information (tensor dimension, type, name and so on) of input data for the given model.
+     * Updates the output information for the model internally.
+     *
+     * @param in The input tensors information
+     *
+     * @throws IllegalStateException if failed to set the input information
+     * @throws IllegalArgumentException if given param is null
+     */
+    public void setInputInfo(@NonNull TensorsInfo in) {
+        checkPipelineHandle();
+
+        if (in == null) {
+            throw new IllegalArgumentException("Input tensor data is null");
```

Comment:
plz update exception msg 'tensor info'

Reply:
`in` is the input tensor data, `inInfo` is the tensor info.